mappings/cvss_v3.json

```diff
@@ -0,0 +1,1031 @@
+{
+  "metadata": {},
+  "content": [
+    {
+      "id": "server_security_misconfiguration",
+      "children": [
+        {
+          "id": "unsafe_cross_origin_resource_sharing",
+          "cvss_v3": "AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N"
+        },
+        {
+          "id": "path_traversal",
+          "cvss_v3": "AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N"
+        },
+        {
+          "id": "directory_listing_enabled",
+          "children": [
+            {
+              "id": "sensitive_data_exposure",
+              "cvss_v3": "AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N"
+            },
+            {
+              "id": "non_sensitive_data_exposure",
+              "cvss_v3": "AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:N"
+            }
+          ]
+        },
+        {
+          "id": "same_site_scripting",
+          "cvss_v3": "AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:N"
+        },
+        {
+          "id": "ssl_attack_breach_poodle_etc",
+          "cvss_v3": "AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N"
+        },
+        {
+          "id": "using_default_credentials",
+          "children": [
+            {
+              "id": "production_server",
+              "cvss_v3": "AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:N"
+            },
+            {
+              "id": "staging_development_server",
+              "cvss_v3": "AV:A/AC:L/PR:N/UI:N/S:U/C:L/I:H/A:N"
+            }
+          ]
+        },
+        {
+          "id": "misconfigured_dns",
+          "children": [
+            {
+              "id": "subdomain_takeover",
+              "cvss_v3": "AV:N/AC:L/PR:N/UI:R/S:U/C:N/I:H/A:L"
+            },
+            {
+              "id": "zone_transfer",
+              "cvss_v3": "TODO"
+            },
+            {
+              "id": "missing_caa_record",
+              "cvss_v3": "TODO"
+            }
+          ]
+        },
+        {
+          "id": "mail_server_misconfiguration",
+          "children": [
+            {
+              "id": "missing_spf_on_email_domain",
+              "cvss_v3": "AV:N/AC:L/PR:N/UI:R/S:U/C:L/I:L/A:N"
+            },
+            {
+              "id": "email_spoofable_via_third_party_api_misconfiguration",
+              "cvss_v3": "AV:N/AC:L/PR:N/UI:R/S:U/C:L/I:L/A:N"
+            },
+            {
+              "id": "missing_spf_on_non_email_domain",
+              "cvss_v3": "AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:N"
```

Comment:
since this is used for most children can we declare it at the parent level and override where necessary?